internal/pkg/selfupdate/update.go

```diff
@@ -55,10 +58,23 @@ func UpgradeInPlace(ctx context.Context, curSV semver.Version, baseSiteURL strin
 		return nil
 	} else {
```

Comment:
How about remove `detach` parameter? It's not working now

Reply:
sure